### ISSUE TITLE
etcdctl is reporting that cluster member is healthy even after it has been terminated

### ISSUE THREAD

**Issue by mariusgrigaitis** [coreos/etcd#2983]
1. Set up cluster of 4 nodes
2. Power off one of the nodes
3. Run etcdctl cluster-health

Result: powered off member is reported as healthy even though it has been powered off 10 minutes ago
Expected: powered off member is reported as unhealthy

```
admin@ip-172-27-0-152:~$ etcdctl member list
14d8d41e1efc933d: name=ip-172-27-0-19 peerURLs=http://172.27.0.19:2380 clientURLs=http://172.27.0.19:2379,http://172.27.0.19:4001
6da328191392fa29: name=ip-172-27-0-124 peerURLs=http://172.27.0.124:2380 clientURLs=http://172.27.0.124:2379,http://172.27.0.124:4001
fa02c5cafaba7aef: name=ip-172-27-0-152 peerURLs=http://172.27.0.152:2380 clientURLs=http://172.27.0.152:2379,http://172.27.0.152:4001
fd831b869aab1d79: name=ip-172-27-0-101 peerURLs=http://172.27.0.101:2380 clientURLs=http://172.27.0.101:2379,http://172.27.0.101:4001
admin@ip-172-27-0-152:~$ ping -c 1 172.27.0.124
PING 172.27.0.124 (172.27.0.124) 56(84) bytes of data.
64 bytes from 172.27.0.124: icmp_req=1 ttl=64 time=0.679 ms

--- 172.27.0.124 ping statistics ---
1 packets transmitted, 1 received, 0% packet loss, time 0ms
rtt min/avg/max/mdev = 0.679/0.679/0.679/0.000 ms
admin@ip-172-27-0-152:~$ ping -c 1 172.27.0.101
PING 172.27.0.101 (172.27.0.101) 56(84) bytes of data.
From 172.27.0.152 icmp_seq=1 Destination Host Unreachable

--- 172.27.0.101 ping statistics ---
1 packets transmitted, 0 received, +1 errors, 100% packet loss, time 0ms

admin@ip-172-27-0-152:~$ time etcdctl cluster-health
cluster is healthy
member 14d8d41e1efc933d is healthy
member 6da328191392fa29 is healthy
member fa02c5cafaba7aef is healthy
member fd831b869aab1d79 is healthy

real	0m5.427s
user	0m0.004s
sys	0m0.000s
admin@ip-172-27-0-152:~$ time etcdctl get /a
b

real	0m9.104s
user	0m0.004s
sys	0m0.000s
admin@ip-172-27-0-152:~$ time etcdctl set /a "foo"
foo

real	0m8.309s
user	0m0.004s
sys	0m0.000s
admin@ip-172-27-0-152:~$ etcdctl -v
etcdctl version 2.0.11
admin@ip-172-27-0-152:~$ etcd --version
etcd Version: 2.0.11
Git SHA: 0678329
Go Version: go1.4.2
Go OS/Arch: linux/amd64
```

**Comment by mariusgrigaitis** [comment]
After restarting etcd deamon on one of the machines, etcdctl starts reporting member as unhealthy. Maybe it is related #2838

**Comment by xiang90** [comment]
@mariusgrigaitis Duplicate with #2888. Fixed by #2906. Please try with current master. Thanks.
